### PR TITLE
Refactor case summary demographics

### PR DESCRIPTION
**Pull request by ljwoodley**
`ljw_uf_stp_report` will give slightly lower counts as I'm filtering to only cases affiliated with UF. If this is incorrect I'll remove the filter from line 65 in this report. 

There is some excess code in the original report that is not needed to create the report. 
@worldwidey Is it ok to move this extra code to another R script or delete as it's a bit confusing. 

If `ljw_uf_stp_report` is  ok with everyone I'll move it to `uf_stp_report` so this PR can be merged.

### CODE REVIEW

File: case_summary_demographics.R [hunk 1]
```diff
@@ -1,197 +1,93 @@
...
-                                                                       ifelse(case_age>=25&case_age<=29,"25-29",
-                                                                              ifelse(case_age>=30&case_age<=34,"30-34",
-                                                                                     ifelse(case_age>=35&case_age<=39,"35-39",
-                                                                                            ifelse(case_age>=40&case_age<=49,"40-49",
-                                                                                                   ifelse(case_age>=50&case_age<=59,"50-59",
-                                                                                                          ifelse(case_age>=60&case_age<=69,"60-69",
-                                                                                                                 ifelse(case_age>=70,"70+","NA"))))))))))))
-  
-  Age_Table<-case_data %>% filter(agegroup!="NA") %>% group_by(agegroup) %>% tally(case_ufaffiliation)
+  mutate_at(vars(contains("date")), as_date) %>%
```
Comment (worldwidey) [inline case_summary_demographics.R:6]: This is a way better efficient way!

**Comment by ChemiKyle** [comment]
Hey @ljwoodley did you mean to include `ljw_uf_stp_report.Rmd` rather than an update to `uf_stp_report.Rmd`?

File: uf_stp_report.Rmd [hunk 2]
```diff
@@ -13,7 +13,13 @@ always_allow_html: yes
...
 ```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = FALSE, warning=FALSE, message = FALSE)
+knitr::opts_chunk$set(
+	echo = FALSE,
+	fig.height = 3.7,
+	fig.width = 5.5,
+	message = FALSE,
+	warning = FALSE
+)
 knitr::opts_chunk$set(fig.width=5.5, fig.height=3.7)
```
Comment (ChemiKyle) [inline uf_stp_report.Rmd:23]:
```suggestion
```
This line is now redundant

**Comment by ChemiKyle** [comment]
A little jarring, the `person_type` pie chart now shouts

image

**Comment by ChemiKyle** [comment]
Reports look the same with the exception of dropping ~10 cases since we now filter for only UF affiliates (which is appropriate given the captions)

I'm going to try to fix the CAPSLOCK on person_type and then I'll merge.

**Pull request merged**